### PR TITLE
add start page

### PR DESCRIPTION
**Pull request by YoohaBae**
- need to fix bug of navigation bar changing itself to logout version when the page refreshes.

### CODE REVIEW

File: src/components/start.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                <header className={"header-wrapper"} id={"about"}>
+                    <div className={"header-solid"}>
+                        <div className={"header-content content"}>
+                            <div className={"header-copy"}>
+                                <h1 className={"header-h1 smaller-text-space"}>Gauss Talk</h1>
+                                <p className>
+                                    A platform where Gaussians can communicate, discuss, and share their different ideas and improve company culture
+                                </p>
+                                <div className={"btn-group text-center"}>
+                                    <Button href="https://login.microsoftonline.com/cfcd9b87-7c5a-4042-9129-abee6253febe/oauth2/v2.0/authorize?client_id=7fc37514-c400-4b28-a6d6-e19a9ae981b6&response_type=code&redirect_uri=http://localhost:3000/auth&scope=User.read" target="_self" variant={"contained"}>
```
Comment (yonsweng) [inline src/components/start.js:28]: redirect_uri is localhost oh no. The link is on the other file too. I had to find it at the first time.

File: src/index.js [hunk 2]
```diff
@@ -7,25 +7,27 @@ import {
...
 import Logout from './components/logout.js'
 import { TokenProvider } from './components/token-context.tsx';
 import 'bootstrap/dist/css/bootstrap.css'
 import './assets/styles/index.css';
 
 
-const Root = () => {
-  return (
+
+const Root = () => (
```
Comment (yonsweng) [inline src/index.js:19]: Good. I'm learning js from you.

File: src/components/custom-bottom-nav-bar.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+class CustomBottomNavbar extends React.Component {
+
+    render() {
+        return (
+            <>
+                <Navbar bg="light" variant="light" sticky={"bottom"}>
+                    <Container>
+                        <Navbar.Brand href="/">
+                            <img
```
Comment (yonsweng) [inline src/components/custom-bottom-nav-bar.js:13]: Looks not so aligned yet. Let's have another PR to fix it unless it's easy to fix right now.

File: src/components/custom-nav-bar.js [hunk 4]
```diff
@@ -12,6 +12,28 @@ class CustomNavbar extends React.Component {
     super(props);
   }
 
+  navlinks(props) {
+    let token = props.token;
+    if(token === null) {
+      return (
+          <>
+            <Nav.Link href="#about">About</Nav.Link>
```
Comment (yonsweng) [inline src/components/custom-nav-bar.js:20]: I think one of the react's advantages is that it makes a SPA (Single Page Application). If you click Nav.Link, I found it refreshes the page. Refreshing makes the application loses `Context` and other things in the memory. So I suggest to use `<Nav.Link as={Link} to="/">`. I found it from https://stackoverflow.com/questions/59530374/react-boostrap-and-react-router-causing-full-page-reload.

**Review by yonsweng — CHANGES_REQUESTED** [review]
LGTM! (except the comments)

File: src/components/start.js (hunk 1 above)
Comment (yonsweng) [reply src/components/start.js:28]: Should be https://talk.gausslabs.ai/home/auth right now.

File: src/components/start.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                                            JOIN&nbsp;NOW
+                                    </Button>
+                                </div>
+                            </div>
+                            <div className={"header-img-wrapper"}>
+                                <img className={"header-img"} src={start_img_1}></img>
```
Comment (yonsweng) [inline src/components/start.js:34]: All img tags should contain the `alt` property. Warnings pop up if not.

**Review by yonsweng — CHANGES_REQUESTED** [review]
There 4 img tags in start.js. Please specify alt properties.

File: src/components/start.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    render() {
+        return (
+            <>
+                <CustomNavbar />
+                <header className={"header-wrapper"} id={"about"}>
+                    <div className={"header-solid"}>
+                        <div className={"header-content content"}>
+                            <div className={"header-copy"}>
+                                <h1 className={"header-h1 smaller-text-space"}>Gauss Talk</h1>
+                                <p className>
```
Comment (yonsweng) [inline src/components/start.js:24]: This gives a warning in the chrome console.

File: src/components/start.js [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </header>
+                <section className={"section"} id={"features"}>
+                    <div className={"section-solid"}>
+                        <div className={"section-content content"}>
+                            <div className={"section-copy"}>
+                                <h1 className={"section-h1"}>Company Chat</h1>
+                                <p className>
```
Comment (yonsweng) [inline src/components/start.js:44]: This gives a warning in the chrome console.

File: src/components/start.js [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+                </section>
+                <section className={"section"}>
+                    <div className={"section-solid"}>
+                        <div className={"section-content content"}>
+                            <div className={"section-img-wrapper"}>
+                                <img className={"section-img"} src={lunch_together}></img>
+                            </div>
+                            <div className={"section-copy"}>
+                                <h1 className={"section-h1"}>Lunch Together</h1>
+                                <p className>
```
Comment (yonsweng) [inline src/components/start.js:62]: This gives a warning in the chrome console.

File: src/components/start.js [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </section>
+                <section className={"section"}>
+                    <div className={"section-solid"}>
+                        <div className={"section-content content"}>
+                            <div className={"section-copy"}>
+                                <h1 className={"section-h1"}>Random 1 : 1</h1>
+                                <p className>
```
Comment (yonsweng) [inline src/components/start.js:74]: This gives a warning in the chrome console.

File: src/components/custom-bottom-nav-bar.js (hunk 3 above)
Comment (YoohaBae) [reply src/components/custom-bottom-nav-bar.js:13]: I want to make a custom navbar without using bootstrap for the bottom but didn't have the time to do it yet.

Comment (yonsweng) [reply src/components/custom-bottom-nav-bar.js:13]: OK. Let's just leave it.

**Pull request merged**